apps/fz_http/test/fz_http_web/live/device_live/unprivileged/index_test.exs

```diff
@@ -1,5 +1,8 @@
 defmodule FzHttpWeb.DeviceLive.Unprivileged.IndexTest do
   use FzHttpWeb.ConnCase, async: false
+  import FzHttp.GatewaysFixtures, only: [setup_default_gateway: 1]
```

Comment:
You can add `setup_default_gateway/1` to `test_helpers.ex` and it will be imported by default, so this line won't be required